### PR TITLE
Fix #401: Add SSL/TLS testing tool from SSLLabs

### LINKED ISSUE #401: SSL Labs Upgrade with new API access
https://www.qualys.com/company/newsroom/news-releases/usa/2015-03-17-qualys-enhances-ssl-labs-service-new-api-access-safeguard-websites/

### PR DESCRIPTION
**Pull request by arunk-s**

### CODE REVIEW

File: install/distro-independent.cfg [hunk 1]
```diff
@@ -79,3 +79,7 @@ command = python2 %(RootDir)s/dictionaries/dict_merger_svndigger_raft.py
 # [Vulnerability Database]
 # directory = ~/.owtf/db/
 # command = python2 %(RootDir)s/scripts/vuln_database.py
+
+[SSLLABS SCAN]
+directory = %(RootDir)s/tools/restricted/ssllabs-scan
+command = wget --tries=3 https://github.com/ssllabs/ssllabs-scan/releases/download/v1.3.0/ssllabs-scan_1.3.0-linux64.tgz; tar xvf *
```
Comment (delta24) [inline install/distro-independent.cfg]: Add here to remove the downloaded  `.tar.gz` file after extracting

**Comment by arunk-s** [comment]
@delta24 PR updated.

**Comment by delta24** [comment]
Working verified but the report is .json format and and very long. Can you generate a nice looking .txt/static HTML report from the JSON?

**Comment by arunk-s** [comment]
@delta24, do you want to me to create a new parsing script which would then generate the information in a text file or maybe use existing ones:
https://github.com/moheshmohan/pyssltest
Also as metasploit framework have already integrated the sslscan and we have metasploit modules running on OWTF via plugins, can't we run sslscan through it (as a plugin) and collect the results ?

**Comment by delta24** [comment]
@arunk-s No, a simple parsing python/shell script can do the job. Metasploit generally runs much slower and Metasploit+ssllabs-scan will be too slow.

**Comment by delta24** [comment]
Fixed in d1bfddfbf1a4d4adbae951676a1febc5b4d9eb42

**Pull request closed without merging**